Jenkinsfile-hotfix

```diff
@@ -164,7 +169,7 @@ pipeline {
             echo "Deploying to hotfix..."
             openshiftTag destStream: 'hotfix-eagle-admin', verbose: 'false', destTag: 'hotfix', srcStream: 'hotfix-eagle-admin', srcTag: "${IMAGE_HASH}"
             sleep 5
-            openshiftVerifyDeployment depCfg: 'hotfix-eagle-admin', namespace: 'esm-test', replicaCount: 1, verbose: 'false', verifyReplicaCount: 'false', waitTime: 600000
+            openshiftVerifyDeployment depCfg: 'hotfix-eagle-admin', namespace: TESTPROJECT, replicaCount: 1, verbose: 'false', verifyReplicaCount: 'false', waitTime: 600000
```

Comment:
is this var missing curly braces?

Reply:
I don't believe so.  I'm not doing string interpolation here so I don't need to do `${TESTPROJECT}`.  In this case I'm just passing in the env var as is.